src/Monolog/Logger.php

```diff
@@ -527,8 +527,11 @@ public static function getLevelName($level)
      */
     public static function toMonologLevel($level)
     {
-        if (is_string($level) && defined(__CLASS__.'::'.strtoupper($level))) {
-            return constant(__CLASS__.'::'.strtoupper($level));
+        if (is_string($level)) {
+            $upper = mb_strtoupper($level);
```

Comment:
`ext-mbstring` should be added to composer.json.

Reply:
Nope.. we should rather not require mb_strtoupper. This can be done in another way like `strtoupper(str_replace('i', 'I', $level))` - hackish but seems to work.

Reply:
Indeed.. May I suggest to use `strtr($level, 'abcdefgilmnortuwy', 'ABCDEFGILMNORTUWY');`which is faster?

List of chars are extract from current level, but have to be updated if new levels are added (test have to cover it)

Reply:
@jderusse I think your suggestion is pretty clever, will try that